src/components/Patient/PatientRegistration.tsx

```
@@ -307,558 +288,476 @@ export default function PatientRegistration(
+                      defaultValue={field.value}
+                    >
+                      <FormControl>
+                        <SelectTrigger>
+                          <SelectValue
+                            placeholder={t("please_select_blood_group")}
+                          />
+                        </SelectTrigger>
+                      </FormControl>
+                      <SelectContent>
+                        {BLOOD_GROUP_CHOICES.map((bg) => (
+                          <SelectItem value={bg.id}>{bg.text}</SelectItem>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)